3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/KinesisEnrich.scala

```diff
@@ -29,7 +29,6 @@ import scala.collection.JavaConverters._
 import scala.io.Source
 import scala.sys.process._
 import scala.util.Try
-
```

Comment:
:art: why were those lines removed? we tend to separate java from scala from other imports